test/test-track.sh

```diff
@@ -292,3 +292,136 @@ begin_test "track blocklisted files with glob"
   grep "Pattern .git\* matches forbidden file" track.log
 )
 end_test
+
+begin_test "track lockable"
+(
+  set -e
+
+  repo="track_lockable"
+  mkdir "$repo"
+  cd "$repo"
+  git init
+
+  # track *.jpg once, lockable
+  git lfs track --lockable "*.jpg" | grep "Tracking \*.jpg"
+  numjpg=$(grep -e "\*.jpg.*lockable" .gitattributes | wc -l)
```

Comment:
If you want to keep the echo/cat/exit pattern below, I'd extract to this to a function for clarity:

```sh
assert_attributes_count() {
  local ext="$1"
  local lockable="$2"
  local expected="$3"
}

# ...

assert_attributes_count "jpg" "lockable" 1
```

(You could use "$lockable" to help compose the pattern and make it a little easier for callers, or define two functions: `assert_lockable_attribs_count`, `assert_lockable_attribs_count`).